app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,13 @@
 class PostsController < ApplicationController
+  around_action :skip_bullet
+
+  def skip_bullet
+    Bullet.enable = false
+    yield
+  ensure
+    Bullet.enable = true
+  end
+
   def index
     @user2 = User.find(params[:user_id])
     @posts = @user2.posts
```

Comment:
- [ ] Let's keep our application scalable by eliminating the N+1 query problems from our application(Seems n extra queries are being executed when fetching all posts and comments for a @user2) Kindly fix these issues.

**Hints:**  We can try to eliminate the issue like this:

@posts = @user2.posts.includes(:comments)